### PR TITLE
[dashboard] update experiments and schedules

### PR DESCRIPTION
**Pull request by fewdan**
Signed-off-by: “fewdan” <fewdan@hotmail.com>

### What problem does this PR solve?

update experiments and schedules

Problem Summary:

### What is changed and how it works?

update experiments and schedules

What's Changed:

### Related changes

* None

### Checklist

Tests

- [ ] Unit test
- [ ] E2E test
- [ ] Manual test (add detailed scripts or steps below)
- [x] No code

Side effects

- [ ] Breaking backward compatibility

### Release note 

```release-note
Please add a release note.
If you don't think this PR needs a release note then fill it with None.
```

### CODE REVIEW

File: pkg/apiserver/schedule/schedule.go [hunk 1]
```diff
@@ -766,7 +765,13 @@ func (s *Service) updateScheduleFun(exp *core.KubeObjectDesc, kubeCli client.Cli
 	sch.SetAnnotations(meta.Annotations)
 
 	var spec v1alpha1.ScheduleSpec
-	mapstructure.Decode(exp.Spec, &spec)
+	bytes, err := json.Marshal(exp.Spec)
```
Comment (cwen0) [inline pkg/apiserver/schedule/schedule.go:768]: Have you test this API?

Comment (fewdan) [reply pkg/apiserver/schedule/schedule.go:768]: yes,  but just manual test

**Review by YangKeao — APPROVED** [review]
LGTM

**Review by cwen0 — APPROVED** [review]
LGTM

**Pull request merged**